docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -1,19 +1,127 @@
+
+For database interaction, you will need to set related properties. They have `db.connection.` prefix followed by the database name (`vividus` in the example). Come up with your own database name to use within the tests. Put properties into the `environment.properties` file:
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:dbms://databaseUrl:port/databaseName?parameter1=true&parameter2=value <1>
+db.connection.vividus.username= <2>
+db.connection.vividus.password= <2>
+db.connection.vividus.driver-class-name=<3>
+----
+<1> Database address. Consists of:
+    - jdbc: - reference to JDBC;
+    - dbms: - Database Management System, e.g. `postgresql`;
+    - //databaseUrl - address to your database;
+    - :port - port number, e.g. 5432;
+    - /databaseName - name of your database;
+    - parameter - any additional parameters you need to pass;
++
+IMPORTANT: URL structure slightly changes depending on the database management system. Below you can find templates for different DBMS
++
+.Click to see URL templates
+[%collapsible]
+====
+*MySQL*
+[source,URL,subs="attributes+"]
+----
+jdbc:mysql://protocol//hosts/database?properties
+----
+*Microsoft SQL*
+[source,URL,subs="attributes+"]
+----
+jdbc:sqlserver://serverName\instanceName:portNumber;property=value;property=value
+----
+*PostgreSQL*
+[source,URL,subs="attributes+"]
+----
+jdbc:postgresql://host:port/database?properties
+----
+*Oracle Database SID*
+[source,URL,subs="attributes+"]
+----
+jdbc:oracle:thin:user/password@host:port:SID
+----
+*Oracle Database Service Name*
+[source,URL,subs="attributes+"]
+----
+jdbc:oracle:thin:<user>/password@//host:port/service
+----
+====
+<2> Credentials for given database connection;
+<3> Driver class name (Optional), e.g. `org.postgresql.Driver`. You might need it to specify the driver name for the exact database.
+
+=== Setup examples
+
+Here are some examples of setting properties for different databases. Replace values in the URL with appropriate ones for your database and then put them into the `*.properties` file to establish a connection.
+
+==== MySQL
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:mysql://mysql.db.server:3306/my_database?useSSL=false&serverTimezone=UTC
+db.connection.vividus.username=vividus
+db.connection.vividus.password=passwordValue
+----
+
+==== Microsoft SQL
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:sqlserver://mssql.db.server\\mssql_instance;databaseName=my_database
+db.connection.vividus.username=vividus
+db.connection.vividus.password=passwordValue
+----
+
+==== PostgreSQL
+
+.environment.properties
+[source,properties,subs="attributes+"]
+----
+db.connection.vividus.url=jdbc:postgresql://postgresql.db.server:5430/my_database?ssl=true&loglevel=2
+db.connection.vividus.username=vividus
+db.connection.vividus.password=passwordValue
+----
```

Comment:
could you please also add:
- CSV (and add a reference to our `Execute SQL against CSV` guide)
- H2 (and add a reference to our `Use in-memory database` guide)
?